cmd/setup/34.sql

```diff
@@ -13,7 +13,7 @@ partition by list (cache_name);
 
 create unlogged table if not exists cache.string_keys(
     cache_name varchar not null check (cache_name <> ''),
-    index_id integer not null check (index_id <> 0),
+    index_id integer not null check (index_id > 0),
```

Comment:
Instead of dropping cockroach support we could create a normal table for cockroach. It isn't that performance but I think it could still be beneficial.